authenticate/config.js

```diff
@@ -0,0 +1,5 @@
+var config = {
+	secret:'xingyunzh-campro-secret'
```

Comment:
We should not store secret like this. It should not be in github.
Please refer to OSS secret mechanism, which stores in environment. The file names "camproKeys.json". You can add an entry there.